src/style.css

```diff
@@ -1 +1,118 @@
-/* Styles go here */
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(fonts/Roboto-Medium.ttf);
+}
+
+.html {
+  box-sizing: border-box;
+
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.body {
+  margin: 0;
+}
+
+.link-reset {
+  color: inherit;
+  text-decoration: none;
+}
+
+.lires {
+  margin: 0;
+  padding: 0;
+
+  list-style-type: none;
+  line-height: 1;
+}
+
+.container {
+  max-width: 1200px;
+
+  margin: 0 auto;
+  padding: 0 50px;
+}
+
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
```

Comment:
BEM is cool, but you can simplify your styles.
This two classes can be combined, also paddings can be set here, and container can be moved outside header